content/en/docs/ambient/install/istioctl-installation/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Install with Istioctl
+description: Install Istio in ambient mode with Istioctl.
+weight: 4
+aliases:
+  - /docs/ops/ambient/install/istioctl-installation
+  - /latest/docs/ops/ambient/install/istioctl-installation
```

Comment:
As I think this is a new document, at least to preliminary.istio.io, we shouldn't need any aliases.